### PR TITLE
[libphonenumber] initial commit of Google's libphonenumber

### PR DESCRIPTION
**Pull request by Redth**

### CODE REVIEW

File: Android/Google.LibPhoneNumber/External-Dependency-Info.txt [hunk 1]
```diff
@@ -0,0 +1,190 @@
+THIRD-PARTY SOFTWARE NOTICES AND INFORMATION
+Do not translate or localize
+
+Xamarin Components for Google's Android Things Support Library incorporates
```
Comment (mattleibow) [inline Android/Google.LibPhoneNumber/External-Dependency-Info.txt]: This is the wrong library name.

File: Android/Google.LibPhoneNumber/nuget/Xamarin.Google.LibPhoneNumber.nuspec [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    <authors>Microsoft</authors>
+    <owners>Microsoft</owners>
+    <requireLicenseAcceptance>true</requireLicenseAcceptance>
+    <summary>Xamarin.Android Bindings for Google LibPhoneNumber</summary>
+    <description>
+      Xamarin.Android Bindings for Google LibPhoneNumber
+    </description>
+    <copyright>© Microsoft Corporation. All rights reserved.</copyright>
+    <projectUrl></projectUrl>
+    <licenseUrl></licenseUrl>
```
Comment (mattleibow) [inline Android/Google.LibPhoneNumber/nuget/Xamarin.Google.LibPhoneNumber.nuspec]: These two things may or may not be correct.

**Review by mattleibow — DISMISSED** [review]

**Comment by mattleibow** [comment]
I like the new SDK-style projects ❤️

File: Android/Google.LibPhoneNumber/build.cake [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+var JAR_VERSION = "8.9.5";
+var JAR_URL = string.Format ("http://search.maven.org/remotecontent?filepath=com/googlecode/libphonenumber/libphonenumber/{0}/libphonenumber-{0}.jar", JAR_VERSION);
+var DOCS_URL = string.Format ("http://search.maven.org/remotecontent?filepath=com/googlecode/libphonenumber/libphonenumber/{0}/libphonenumber-{0}-javadoc.jar", JAR_VERSION);
+var JAR_DEST = "./externals/libphonenumber.jar";
+
+var buildSpec = new BuildSpec () {
+	Libs = new [] {
+		new DefaultSolutionBuilder {
+			SolutionPath = "./Google.LibPhoneNumber.sln",
```
Comment (newky2k) [inline Android/Google.LibPhoneNumber/build.cake]: This is the wrong name for the solution file

File: Android/Google.LibPhoneNumber/build.cake [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+			OutputFiles = new [] { 
+				new OutputFileCopy {
+					FromFile = "./source/bin/Release/Xamarin.Google.LibPhoneNumber.dll",
```
Comment (newky2k) [inline Android/Google.LibPhoneNumber/build.cake]: The output goes into a monodroid sub folder, not this location

**Review by newky2k — CHANGES_REQUESTED** [review]

**Review by newky2k — APPROVED** [review]

**Pull request merged**